src/components/AssistantFormInputs.vue

```diff
@@ -57,6 +57,12 @@
 		</div>
 	</div>
 	<div v-else class="assistant-inputs">
+		<div v-if="selectedTaskTypeId === 'OCA\\ContextChat\\TextProcessing\\ContextChatTaskType'" class="assistant-inputs">
+			<NcCheckboxRadioSwitch :checked.sync="sccEnabled">
+				{{ t('assistant', 'Reduce context') }}
+			</NcCheckboxRadioSwitch>
+			<ContextChatInputForm v-if="sccEnabled" v-model="sccData" />
```

Comment:
```suggestion
			<ContextChatInputForm v-if="sccEnabled" :scc-data.sync="sccData" />
```
But maybe you had good reasons to use v-model.

Reply:
not really, would switch to this for consistency. Just discovered that they do the same thing.